### PR TITLE
EZP-31250: Set text alignment in tables as top

### PR DESCRIPTION
**Pull request by lucasOsti**
| Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZP-31250
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Set text alignment in tables as top

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

**Comment by inakijv** [comment]
Thank you @lucasOsti, Could you please add screenshots?

**Comment by dew326** [comment]
Please update the code styling. Probably changed from `4 spaces` to `1 tab`. Please also check if your Prettier is styling the twig files and if so please disable it on html. We do not use Prettier on html.

File: src/bundle/Resources/views/themes/admin/content/tab/locations/tab.html.twig [hunk 1]
```diff
@@ -13,70 +13,58 @@
...
-                <th>{{ 'tab.locations.path'|trans()|desc('Path') }}</th>
-                <th>{{ 'tab.locations.subitems'|trans()|desc('Sub-items') }}</th>
-                <th colspan="2">{{ 'tab.locations.visibility'|trans()|desc('Visibility') }}</th>
-                <th>{{ 'tab.locations.main'|trans()|desc('Main') }}</th>
-            </tr>
-            </thead>
-            <tbody>
-            {% if locations is not empty %}
-                {% for location in locations %}
+        <div class="ez-scrollable-table-wrapper">
```
Comment (dew326) [inline src/bundle/Resources/views/themes/admin/content/tab/locations/tab.html.twig]: The previous indentation was correct.

File: src/bundle/Resources/views/themes/admin/content/tab/locations/tab.html.twig [hunk 2]
```diff
@@ -85,25 +73,14 @@
 } %}
 
 {% macro table_header_tools(form_add, form_remove, can_add) %}
-    <button
-            data-udw-config="{{ ez_udw_config('single', {}) }}"
-            class="btn btn-primary btn--udw-add"
-            title="{{ 'tab.locations.action.add'|trans|desc('Add Location') }}"
-            {{ not can_add ? 'disabled' }}>
+    <button data-udw-config="{{ ez_udw_config('single', {}) }}" class="btn btn-primary btn--udw-add" title="{{ 'tab.locations.action.add'|trans|desc('Add Location') }}" {{ not can_add ? 'disabled' }}>
```
Comment (GrabowskiM) [inline src/bundle/Resources/views/themes/admin/content/tab/locations/tab.html.twig]: multiline is much better here, having each property in new line

File: src/bundle/Resources/views/themes/admin/content/tab/locations/tab.html.twig [hunk 3]
```diff
@@ -85,25 +73,14 @@
...
     {% set modal_data_target = 'delete-content-types-modal' %}
-    <button
-            id="delete-locations"
-            type="button"
-            class="btn btn-danger"
-            disabled
-            data-toggle="modal"
-            data-target="#{{ modal_data_target }}"
-            title="{{ 'tab.locations.action.delete'|trans|desc('Delete Location') }}">
+    <button id="delete-locations" type="button" class="btn btn-danger" disabled data-toggle="modal" data-target="#{{ modal_data_target }}" title="{{ 'tab.locations.action.delete'|trans|desc('Delete Location') }}">
```
Comment (GrabowskiM) [inline src/bundle/Resources/views/themes/admin/content/tab/locations/tab.html.twig]: ^

**Comment by inakijv** [comment]
Thank you @lucasOsti Could you please add some images?

**Comment by lucasOsti** [comment]
Zrzut ekranu 2020-01-10 o 09 20 12

**Pull request merged**